packages/core/src/definitions/helpers/safe-translate/index.ts

```diff
@@ -0,0 +1,20 @@
+import { useTranslate } from "@hooks/translate";
```

Comment:
Can we use like below?
```suggestion
import type { useTranslate } from "@hooks/translate";
```

Reply:
Not needed actually, but updated them anyways.